src/main/scala/com/yotpo/metorikku/metric/Metric.scala

```diff
@@ -117,28 +133,52 @@ case class Metric(configuration: Configuration, metricDir: File, metricName: Str
     }
   }
 
+  private def getEpochLagTime(dataFrame: DataFrame, lagReporterTimeColumn: Option[String]): Option[Long] ={
```

Comment:
This actually returns the maximum time inside the dataframe, the function name doesn't reflect this